### PR TITLE
mongodb: Added paging parameter in the find procedure

### PR DESCRIPTION
**Pull request by stdob**

### CODE REVIEW

File: src/main/java/apoc/mongodb/MongoDB.java [hunk 1]
```diff
@@ -70,9 +70,17 @@ private Coll getColl(@Name("host") String hostOrKey, @Name("db") String db, @Nam
...
+    @Description("apoc.mongodb.find(host-or-port,db-or-null,collection-or-null,query-or-null,projection-or-null,sort-or-null,pagination,[compatibleValues=true|false]) yield value - perform a find,project,sort operation on mongodb collection")
+    public Stream<MapResult> find(@Name("host") String hostOrKey, 
+                                  @Name("db") String db, 
+                                  @Name("collection") String collection, 
+                                  @Name("query") Map<String, Object> query, 
+                                  @Name("project") Map<String, Object> project,
+                                  @Name("sort") Map<String, Object> sort, 
+                                  @Name(value = "compatibleValues", defaultValue = "false") boolean compatibleValues,
+                                  @Name(value = "skip", defaultValue = "0") Long skip,
+                                  @Name(value = "limit", defaultValue = "0") Long limit) {
```
Comment (jexp) [inline src/main/java/apoc/mongodb/MongoDB.java]: default value should be -1

File: src/main/java/apoc/mongodb/MongoDBColl.java [hunk 2]
```diff
@@ -114,10 +114,12 @@ public long count(Map<String, Object> query) {
     }
 
     @Override
-    public Stream<Map<String, Object>> find(Map<String, Object> query, Map<String, Object> project, Map<String, Object> sort) {
+    public Stream<Map<String, Object>> find(Map<String, Object> query, Map<String, Object> project, Map<String, Object> sort, Long skip, Long limit) {
         FindIterable<Document> documents = query == null ? collection.find() : collection.find(new Document(query));
         if (project != null) documents = documents.projection(new Document(project));
         if (sort != null) documents = documents.sort(new Document(sort));
+        if (skip != null) documents = documents.skip(skip.intValue());
+        if (limit != null) documents = documents.limit(limit.intValue());
```
Comment (jexp) [inline src/main/java/apoc/mongodb/MongoDBColl.java]: the default will be 0 not null

File: src/main/java/apoc/mongodb/MongoDBColl.java [hunk 3]
```diff
@@ -114,10 +114,12 @@ public long count(Map<String, Object> query) {
     }
 
     @Override
-    public Stream<Map<String, Object>> find(Map<String, Object> query, Map<String, Object> project, Map<String, Object> sort) {
+    public Stream<Map<String, Object>> find(Map<String, Object> query, Map<String, Object> project, Map<String, Object> sort, Long skip, Long limit) {
         FindIterable<Document> documents = query == null ? collection.find() : collection.find(new Document(query));
         if (project != null) documents = documents.projection(new Document(project));
         if (sort != null) documents = documents.sort(new Document(sort));
+        if (skip != null) documents = documents.skip(skip.intValue());
```
Comment (jexp) [inline src/main/java/apoc/mongodb/MongoDBColl.java]: default will be 0 noot null

File: src/main/java/apoc/mongodb/MongoDB.java (hunk 1 above)
Comment (stdob) [reply src/main/java/apoc/mongodb/MongoDB.java]: A limit() value of 0 (i.e. .limit(0)) is equivalent to setting no limit.

https://docs.mongodb.com/manual/reference/method/cursor.limit/#cursor.limit

File: docs/overview.adoc [hunk 4]
```diff
@@ -328,10 +328,10 @@ Adding on config the parameter `failOnError:false` (by default `true`), in case
 
 [cols="3m,2"]
 |===
-| CALL apoc.mongodb.get(host-or-port,db-or-null,collection-or-null,query-or-null,[compatibleValues=true|false]) yield value | perform a find operation on mongodb collection
+| CALL apoc.mongodb.get(host-or-port,db-or-null,collection-or-null,query-or-null,[compatibleValues=true\|false]) yield value | perform a find operation on mongodb collection
```
Comment (jexp) [inline docs/overview.adoc]: @stdob would it make sense to add it here too?

File: src/main/java/apoc/mongodb/MongoDB.java (hunk 1 above)
Comment (jexp) [reply src/main/java/apoc/mongodb/MongoDB.java]: Ah ok. :)

File: docs/overview.adoc (hunk 4 above)
Comment (stdob) [reply docs/overview.adoc]: @jexp Yes, without escaping it broke the layout.

Comment (jexp) [reply docs/overview.adoc]: You can also just use `/` instead :)

Comment (jexp) [reply docs/overview.adoc]: I meant skip/limit to the `get` procedure.

**Pull request closed without merging**